src/com/projectkorra/projectkorra/Element.java

```diff
@@ -96,12 +96,31 @@ public Element(String name, ElementType type) {
 	 * @param type ElementType specifies if its a regular element or chi style
 	 *            element.
 	 * @param plugin The plugin that is adding the element.
+	 * @param color Element's color
+	 * @param prefix Element's prefix
 	 */
-	public Element(String name, ElementType type, Plugin plugin) {
+	public Element(String name, ElementType type, Plugin plugin, String color, String prefix) {
 		this.name = name;
 		this.type = type;
 		this.plugin = plugin;
 		ALL_ELEMENTS.put(name.toLowerCase(), this);
+		if(getElement(name) instanceof SubElement) {
+			String mainName = prefix;
+			if(((ConfigManager.languageConfig.get().getString("Chat.Colors." + mainName + "Sub")) == null)) {

```

Comment:
I could make per SubElement colors, but I would need to change the way you're saving SubElements in the configuration file.It'll be like
`  Colors:
    Avatar: DARK_PURPLE
    Air: GRAY
    Water: AQUA
    Earth: GREEN
    Fire: RED
    Chi: GOLD
    SubElements:
          Flight: DARK_GRAY
          Lightning: DARK_RED`
and so on